frontend/components/commons/sidebar/SidebarMenu.vue

```diff
@@ -17,23 +17,21 @@
 
 <template>
   <div class="sidebar">
-    <span v-for="group in sidebarGroups" :key="group">
-      <div class="sidebar__info">
-        <p>{{ group }}</p>
-        <sidebar-button
-          v-for="button in filteredSidebarItems.filter(
-            (button) => button.group === group
-          )"
-          :id="button.id"
-          :key="button.id"
-          :active-view="[viewMode, currentMetric]"
-          :icon="button.icon"
-          :tooltip="button.tooltip"
-          :type="group"
-          @button-action="action(button.action, button.id)"
-        />
-      </div>
-    </span>
+    <div v-for="group in sidebarGroups" :key="group" class="sidebar__info">
+      <p>{{ group }}</p>
+      <sidebar-button
+        v-for="button in filteredSidebarItems.filter(
+          (button) => button.group === group
+        )"
```

Comment:
performance : create a computed for `filteredSidebarItems.filter(
          (button) => button.group === group
        )"`